evfl/enums.py

```diff
@@ -71,3 +71,8 @@ class BuildResultType(enum.IntEnum):
     kInvalidOperation = 1
     kResFlowchartNotFound = 2
     kEntryPointNotFound = 3
+
+class VariableType(enum.IntEnum):
```

Comment:
This is actually the same enum as ContainerDataType (aka `ore::ResMetaData::DataType`)